### DISCUSSION TITLE
What shotcuts to use to open such structure of windows?

### DISCUSSION THREAD

**Discussion by sergeda** [regolith-linux/regolith-desktop#1019] [Q&A] [answered]
I want to have two terminal windows one above other occupying whole width and under them two more terminal windows split vertically each occupying 50% of the width of the screen. Is it possible to open them just with standard shortcuts? If yes, what combinations should I use?

**Comment by ajeshp** [comment, marked as answer]
`Mod + Enter`                   (Open one (1) terminal)
`Mod + Backspace`          (Toggle Window Orientation to Vertical Split)
`Mod + Enter`                   (Open another (2) terminal)
`Mod + Enter`                   (Open another (3) terminal)
`Mod + Backspace`          (Toggle Window Orientation to Horizontal Split)
`Mod + Enter`                   (Open another (4) terminal)
You should end up with 
      `      1       `
      `      2       `
 `3    ` `    4`

**Reply by sergeda** [reply]
Wow. Works like a charm
